src/CppUTest/JUnitTestOutput.cpp

```diff
@@ -241,26 +250,39 @@ void JUnitTestOutput::writeTestGroupToFile()
 
 // LCOV_EXCL_START
 
```

Comment:
Actually, I had to exclude this from test coverage because they currently serve no useful purpose. Nice to see these put to good use ;-)

In your version, the // LCOV... comments can (and should) be removed